src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
```

Comment:
According to the SOLID principles and checklist, classes should be simple and focused on a single problem. This class should only handle the 'Return' operation logic, and it does correctly. However, make sure that it is being called in the right place in the application flow, which should be from the `main()` method in the `Main` class, not directly within services.